src/Microsoft.AspNet.OData.Shared/Formatter/ODataModelBinderSettings.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.OData.Formatter
+{
+    /// <summary>
+    /// This class describes the settings to use during model binding.
+    /// </summary>
+    public class ODataModelBinderSettings : IODataModelBinderSettings
+    {
+        private bool _enableCaseInsensitiveModelBinding;
+
+        /// <summary>
+        /// Instantiates a new instance of the <see cref="ODataModelBinderSettings"/> class
+        /// and initializes the default settings.
+        /// </summary>
+        public ODataModelBinderSettings()
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether request body binding should be case insensitive.
+        /// </summary>
+        /// <value>The default value is <c>false</c>.</value>
```

Comment:
I'm not sure if this needs to be explicitly stated in the comment, since it is almost obvious that `bool ` variable type defaults to `false `in most programming languages including C#.

Reply:
#Resolved